src/App.tsx

```diff
@@ -1,26 +1,219 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable */
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { USER_ID, createTodos, deleteTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+
+export enum TodoStatus {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
 
-const USER_ID = 0;
 
 export const App: React.FC = () => {
+  const [error, setError] = useState<string | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [filterStatus, setFilterStatus] = useState<TodoStatus>(TodoStatus.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingTodoIds, setLoadingTodoIds] = useState<number[]>([]);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (filterStatus) {
+      case TodoStatus.Active:
+        return !todo.completed;
+      case TodoStatus.Completed:
+        return todo.completed;
+      default:
+        return true;
+    }
+  });
+
+  useEffect(() => {
+    if (USER_ID) {
+      handleRequest();
+    }
+
+    setTimeout(() => {
+      setError('');
+    }, 3000);
+  }, []);
+
+  // Відправляє запит на сервер для отримання списку (todos).
+  const handleRequest = async () => {
+    try {
+      const allTodo = await getTodos();
+
+      setTodos(allTodo);
+      setError(null);
+    } catch (errors) {
+      setError('Unable to load todos');
+    }
+  };
+
+  const isLoading = !!loadingTodoIds.length;
+
+  const inputRef = useRef<HTMLInputElement>(null);
+  // додавання фокусу на input
```

Comment:
If you need comments in the code, they should be in English only. Fix it everywhere